src/Microsoft.Language.Xml/Syntax/XmlAttributeSyntax.cs

```diff
@@ -111,6 +111,74 @@ public string Value
             }
         }
 
+        /// <summary>
+        /// Get Normalized <see cref="Value"/> of <see cref="XmlStringSyntax"/>
+        /// </summary>
+        /// <remarks>
+        /// Normalization specs:
+        /// <seealso href="https://www.w3.org/TR/2006/REC-xml11-20060816/#sec-line-ends">2.2.12 [XML] Section 3.3.3</seealso/>
+        /// <seealso href="https://learn.microsoft.com/en-us/openspecs/ie_standards/ms-xml/389b8ef1-e19e-40ac-80de-eec2cd0c58ae">2.11 [XML} End-of-Line Handling</seealso/>
+        /// </remarks>
+        public string NormalizedValue

```

Comment:
I've been thinking more about this and I think we should just return the normalized value from the `Value` property, instead of introducing the NormalizedValue. Value is something that is sitting "on top" of the parse tree, it's higher level and expected to be passed to the users. I think users would expect to see the normalized value here. They can always call Attribute.GetText() to get the underlying text in the file.